### ISSUE TITLE
Fix MS Access rResources4Wiki report

### ISSUE THREAD

**Issue by darozak** [Darwin-River/Ex-Machinis#239]
This report creates a wiki-formatted table for use in reproducing resource data in the wiki.  For some reason, it's rendering all numeric values (resource ID and mass) as zeros.  I need to fix this.

**Comment by darozak** [comment]
The query properly displays the resource ID but it is rendered as a zero in the report.  I tried recreating the report from qQueries4Wiki, which properly displays query IDs.  However, it also shows zeros for resource ID when converted.

The only difference I can find between queries and resources tables is that the latter displays "#Deleted" for several entries:
![image](https://user-images.githubusercontent.com/33500329/182837913-426a8899-1e84-4909-9e27-6ebb041ac2b6.png)

Searching online in help forums, it looks as if this issue might be caused when the table key is too big for MS Access to handle.  When I checked, I found that resource ID was int(10) compared to queries ID which was only int(5).  I redefined resource ID as int(5) but this didn't appear to fix the problem.

**Comment by darozak** [comment]
I also tried relinking the table without any success.

**Comment by darozak** [comment]
I think I've figured out away to forgo use of the Access database and rely on the economy Excel file instead.  I already use the economy spreadsheets to program new content for the game.  Now I've started modifying the spreadsheets so that they automatically generate wiki tables from the data (https://github.com/Darwin-River/Ex-Machinis/commits/develop).  As a result it looks like I no longer have a need for the access database.

I'm closing this issue because it's being resolved by mods to the economy spreadsheets.
